core/unix/loader.c

```diff
@@ -1574,7 +1574,7 @@ dynamorio_lib_gap_empty(void)
         while (memquery_iterator_next(&iter) && iter.vm_start < dr_end) {
             if (iter.vm_start >= dr_start && iter.vm_end <= dr_end &&
                 iter.comment[0] != '\0' &&
-                strstr(iter.comment, DYNAMORIO_LIBRARY_NAME) == NULL) {
+                strstr(iter.comment, get_dynamorio_library_path()) == NULL) {
```

Comment:
I worry that this won't get inlined or sthg inside this loop -- maybe worth explicitly putting into a local?

Reply:
Done